console/console-init/ui/src/modules/project/components/IoTReview/IoTReview.tsx

```diff
@@ -148,7 +148,8 @@ export const IoTReview: React.FunctionComponent<IIoTReviewProps> = ({
               mode="xml"
               theme="github"
               fontSize={14}
-              onChange={() => {}}
+              width={"auto"}
+              // onChange={() => {}}
               value={"data"}
```

Comment:
do we need hard code value "data"  ?

Reply:
It will be removed once we implement query to get yaml config from server